atum/src/main/scala/za/co/absa/atum/utils/S3LocationUtils.scala

```diff
@@ -0,0 +1,42 @@
+package za.co.absa.atum.utils
+
+object S3LocationUtils {
+
+  // hint: https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html#bucketnamingrules
+  val S3LocationRx = "(s3[an]?)://([-a-z0-9.]{3,63})/(.*)".r
```

Comment:
Is this supposed to be public?

Reply:
I thought it could be, it might be useful to others, but I am willing to be persuaded.

Reply:
Sure thing. Please add a small comment and type annotation.

Reply:
Thanks, done.